ResearchKit/Common/ORKContinueButton.m

```diff
@@ -63,17 +69,40 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection {
     if (self.traitCollection.verticalSizeClass != previousTraitCollection.verticalSizeClass) {
         [self updateConstraintConstantsForWindow:self.window];
     }
+    
+    NSString *themeName = [[CEVRKTheme sharedTheme] themeName];
+    if (!themeName || themeName.length == 0) {
+        return;
+    }
+    if ([themeName isEqualToString:kThemeAllOfUs] && self.traitCollection.preferredContentSizeCategory != previousTraitCollection.preferredContentSizeCategory) {
```

Comment:
If you write this as `[kThemeAllOfUs isEqualToString:themeName]`, you can eliminate the `if (!themeName || themeName.length == 0)` check above